cmd/cli/plugin/feature/list.go

```diff
@@ -7,187 +7,206 @@ import (
 	"context"
 	"fmt"
 
-	apierrors "k8s.io/apimachinery/pkg/api/errors"
-	crClient "sigs.k8s.io/controller-runtime/pkg/client"
-
 	"github.com/spf13/cobra"
 
+	corev1alpha2 "github.com/vmware-tanzu/tanzu-framework/apis/core/v1alpha2"
 	"github.com/vmware-tanzu/tanzu-framework/cli/runtime/component"
 	"github.com/vmware-tanzu/tanzu-framework/featuregates/client/pkg/featuregateclient"
 )
 
-var featuregate, outputFormat string
-var activated, deactivated, unavailable, extended bool
+var (
+	featuregate, outputFormat     string
+	activated, deactivated        bool
+	extended, includeExperimental bool
+)
 
-// FeatureListCmd is for activating Features
+// FeatureListCmd is for listing features in (a) featuregate(s).
 var FeatureListCmd = &cobra.Command{
 	Use:   "list",
-	Short: "List Features",
+	Short: "List features",
 	Args:  cobra.NoArgs,
 	Example: `
-	# List a clusters Features
+	# List feature(s) referenced in featuregate(s)
```

Comment:
we would also list features that are not part of featuregate(s), so i this can be reworded